timescaledb/tutorials/nfl-analytics/ingest-and-query.md

```diff
@@ -199,31 +199,35 @@ print("Inserting scores.csv")
 insert("data/scores.csv", "scores")
 
 # iterate over each week's CSV file and insert them
-for i in range(1, 18): 
+for i in range(1, 18):
     print(f'Inserting week{i}'.format(i = str(i)))
     insert(f'data/week{i}.csv'.format(i=i), "tracking")
-    
+
 conn.close()
 ```
 
 ## Run your first query
 
-Now that you have all the data ingested, you can run the first aggregation query 
+Now that you have all the data ingested, you can run the first aggregation query
 and examine the results. For most of the example queries in this tutorial,
-you'll need to aggregate data from the `tracking` table, which 
-contains multiple rows per player for each play (because the data is sampled 
+you'll need to aggregate data from the `tracking` table, which
+contains multiple rows per player for each play (because the data is sampled
 multiple times per second during each play)
 
+<highlight type="important">
+These queries are examples of hyperfunctions. To access hyperfunctions, you need to have installed the  [Timescale toolkit](/how-to-guides/install-timescaledb-toolkit) before you begin.
+</highlight>
```

Comment:
@jacobprall and/or @ryanbooz What am I doing wrong on this one? It still seems to be giving me 404 (the other one is fine).

Reply:
@jacobprall can you help here?